_includes/header.html

```diff
@@ -2,9 +2,7 @@
   <div class="container sidebar-sticky">
     <div class="sidebar-about">
       <h1>
-        <a href="/">
-          {{ site.title }}
-        </a>
+        <a href="/">{{ site.title }}</a>

```

Comment:
Please revert the changes not related to the feature.

Reply:
Reverted. The changes relate to the code-formatting, it would be nice to have an explicit formatting configuration, to avoid divergences.